src/sentry/flags/docs/api.md

```diff
@@ -31,7 +31,7 @@ Retrieve a collection of flag logs.
 
 | Column           | Type   | Description                                          |
 | ---------------- | ------ | ---------------------------------------------------- |
-| action           | string | Enum of `created` or `modified`.                     |
+| action           | string | Enum of `created`, `updated`, or `deleted`.          |
```

Comment:
the response examples below also should be updated!

Reply:
Done!  Also, I renamed the columns to remove the `modified_` prefix and replace it with `created_`.  Where "created" refers to the row itself and not the flag.  I think this will be more reasonable long term.